tests/test_client.py

```diff
@@ -1056,6 +1056,21 @@ def test_chained_exceptions_with_explicit_cause(self):
             }
         ]
 
+    def test_notes(self):
+        e = Exception("exception")
+        e.add_note("exception note 1")
```

Comment:
This fails on Python < 3.11 because `add_note` doesn't exist: https://github.com/bugsnag/bugsnag-python/actions/runs/5444670041/jobs/9903374958#step:5:227

You'll need to skip the test on those versions like this: https://github.com/bugsnag/bugsnag-python/blob/2d1db012fb5a87dce94f71d1fa395865d13da389/tests/test_client.py#L572-L575